src/og-10-CF18-for-cdl.cdl

```diff
@@ -0,0 +1,176 @@
+netcdf file:/Users/sevadjian/projects/og-netcdf/2022_11_test/og-10-CF18-for-cdl.nc {
+  dimensions:
+    N_PARAMS = 3;
+    PARAM_STRLEN = 16;
```

Comment:
If `TRAJECTORY` is a string, we don't need the array size. We are already committed to NetCDF-4.

Reply:
@castelao - Yes, according to the CF docs there are a few ways to handle the dtypes and dimensions depending on which format the group has chosen. I followed the CF-1.8 single trajectory spec in my example but it sounds like this is not what you all had in mind. I recommend that you all generate a file as you have envisioned and then run it through a compliance checker to verify it meets CF-1.8. Once you have that in hand you can confidently add it as an example in the code repo as an .nc and as a .cdl and update the document with more details on the structure you have chosen with references to the CF docs to provide a solid backing for your format. A new pull request has been started for this and I believe @justinbuck has agreed to finish out a NOC .nc file and run it through a checker.

Reply:
So if I understand this well, CF-1.10 allows not using *_STRLEN while this is not the case for CF1-8.
This evolution is simplifying the format.

But there is no CF compliant checker for CF-1.10.

What is the benefit of sticking to 1.8 compared to 1.10 apart from the checker ?
File visualisation? ERRDAP ?

Can we be cf 1.10 and use *_STRLEN at the same time ? so that we are compliant with both ?